docs/grfn_spec.md

```diff
@@ -210,35 +319,41 @@ JSON attribute-value list, with the following schema definition:
 
     <grfn_spec>[attrval] ::=
         "start": <string>
-        "name" : <string>
-        "dateCreated" : <string>
+        "source" : <string>
+        "date_created" : <string>
+        "identifiers" : list of <identifier_spec>
         "functions" : list of <function_spec>
 
+There will be one GrFN spec file corresponding to each source code file
```

Comment:
I wonder whether it's necessary to require a separate GrFN file for each individual source file.  Instead, would it be OK to have a single GrFN spec for a program?  E.g.: from earlier discussions about the SimpleModular code I had the impression that @pauldhein would prefer to have a single GrFN file instead of four separate files corresponding to the four different source files.

Reply:
I agree with @skdebray on this point. I think it is more natural to have a GrFN spec correspond to a program. This also helps shed light on the issue that some programs have no single "start" point and indeed may have no start point at all (unless you consider any subroutine as a start point).

Reply:
Good points. Here is what I've changed the text to (for both the "source" and "start" attributes):

>There may be a single GrFN spec file for multiple source code files.
>    FOR NOW: The main target of a GrFN spec file is all of the source code files involved in defining a program.
>    For this reason, the \"source\" attribute is a list of one or more source code files. The string identifying a source file is represented the same way as a `<namespace_path>` (as described above), except that the final name (the name of the file itself) _will_ include the file extension.
>
> It is also the case that there may be multiple \"start\" points (or none at all) for a given program. For this reason, the \"start\" attribute is a list of zero or more names of the entry point(s) of the (Fortran) source code (for example, the PROGRAM module). In the absence of any entry point, this value will be an empty list: `[]`.

Reply:
Here is the updated `<grfn_spec>` top level:

```
    <grfn_spec>[attrval] ::=
        "source" : list of <string>
        "start": list of <string>
        "date_created" : <string>
        "identifiers" : list of <identifier_spec>
        "functions" : list of <function_spec>
```